oa4mp/serve.go

```diff
@@ -306,6 +306,27 @@ func ConfigureOA4MP() (launcher daemon.Launcher, err error) {
 		return
 	}
 
+	// Ensure the OA4MP storage directory exists and has correct permissions and ownership
+	OA4MPStoragePath := "/opt/scitokens-server/var/storage"
```

Comment:
Is it possible to make this filepath configurable? Presumably OA4MP somehow knows to use this path, so if it's a path Pelican tells it to use (as opposed to it being hardcoded by OA4MP or in a Dockerfile), it'd be better to add this as a configuration option and document what it's for and what gets stored here in `docs/parameters.yaml`. I get a little nervous seeing undocumented posix filepaths hardcoded in packages like this. For example, if this the stuff that gets stored here needs to persist between Docker container restarts, users will have a heck of a time figuring that out unless they go grepping through our source code.

Reply:
The OA4MPStoragePath is a fixed directory controlled by OA4MP. Pelican doesn't tell it to do anything other than changing the permissions here. So I think there is no necessity to make it configurable.